app/src/main/java/org/mozilla/tv/firefox/MainActivity.kt

```diff
@@ -135,12 +134,12 @@ class MainActivity : LocaleAwareAppCompatActivity(), OnUrlEnteredListener, Media
 
     override fun onStart() {
         super.onStart()
-        serviceLocator.pocket.startBackgroundUpdates()
+        serviceLocator.pocketRepo.startBackgroundUpdates()
```

Comment:
I wonder if there is a need for another ViewModel here; MainActivityViewModel? May work with `CursorDispatcher` or any other lifecycle subscriptions that's more on the application level

Reply:
Yeah, that makes sense to me.  Out of scope for this refactor, but I'll add a note on this line to move it to an eventual VM.